include/dlaf/eigensolver/bt_band_to_tridiag.h

```diff
@@ -48,9 +49,9 @@ namespace dlaf::eigensolver {
 // @pre band_size is a divisor of mat_hh.blockSize().cols()
 // @pre mat_e is not distributed
 // @pre mat_hh is not distributed
-template <Backend backend, Device device, class T>
-void backTransformationBandToTridiag(const SizeType band_size, matrix::Matrix<T, device>& mat_e,
-                                     matrix::Matrix<const T, device>& mat_hh) {
+template <Backend B, Device D, class T>
+void backTransformationBandToTridiag(const SizeType band_size, matrix::Matrix<T, D>& mat_e,
+                                     matrix::Matrix<const T, Device::CPU>& mat_hh) {
```

Comment:
```suggestion
                                     matrix::Matrix<const T, D>& mat_hh) {
```
? Or is this one of the temporary requirements until a full GPU implementation is there?

Reply:
Yes. I don't know the plan for it (e.g. if the tridiagonal eigensolver will be fully on GPU or it will be hybrid and the results will still be on CPU), but for the moment we are sure they are on CPU.

@rasolca can detail more about plan/forecast.